inference-engine/tests/functional/plugin/gna/pass_tests/concat_memory.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
I'm not sure if this test is related to pass tests. It doesn't test any passes but reservation of memory for layers.

Reply:
moved the tests to the shared scope